_install-and-configure/configuring-opensearch/cluster-settings.md

```diff
@@ -96,6 +96,8 @@ OpenSearch supports the following cluster-level routing and shard allocation set
     - `REPLICA_FIRST` – Replica shards are relocated first, before primary shards. This prioritization may help prevent a cluster's health status from going red when carrying out shard relocation in a mixed-version, segment-replication-enabled OpenSearch cluster. In this situation, primary shards relocated to OpenSearch nodes of a newer version could try to copy segment files to replica shards on an older version of OpenSearch, which would result in shard failure. Relocating replica shards first may help to avoid this in multi-version clusters. 
     - `NO_PREFERENCE` – The default behavior in which the order of shard relocation has no importance. 
 
+- `cluster.allocator.gateway.batch_size` (Integer): Limits the number of shards sent in one batch to data nodes for fetching metadata. Default is `2000`.
+- `cluster.allocator.existing_shards_allocator.batch_enabled` (Boolean): Enables batch allocation of unassigned shards already existing on the disk. This allows for sending all `Unassigned`shards to the `ExistingShard` allocator. The allocator decides how to allocate the unassigned shards. Default is `False`.
```

Comment:
Enables batch allocation of unassigned shards already existing on the disk as opposed to one shard at a time. This reduces the memory and transport overhead by fetching unassigned shard metadata in a batch call. Default is `False`.

Reply:
Updated the wordings.